app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::Base
+  include DeviseTokenAuth::Concerns::SetUserByToken
+  protect_from_forgery with: :exception, if: :verify_api
```

Comment:
`protect_from_forgery with: :null_exception`? Why did you chose the exception? and why if `verify_api`?